journal/modernized/14-chapter-14.adoc

```diff
@@ -406,27 +458,42 @@ Even though we are an isolated body, not in correspondence with any other Yearly
 yet his everlasting arm will be underneath for our support,
 as we are faithful to Him who has called us to glory, and to virtue.
 
+[.signed-section-closing]
 Your affectionate father,
 
+[.signed-section-signature]
 Joseph Edgerton.
 
+--
+
 [.asterism]
 '''
```

Comment:
hmmm. i think i might have nixed this asterism too. because, things like signatures and opening contexts seem to do a good job conveying that a new letter is beginning, so the asterism feels unnecessary. but i'm open to input here...